Modules/_ctypes/callbacks.c

```diff
@@ -155,20 +160,17 @@ static void _CallPythonObject(void *mem,
 
     assert(PyTuple_Check(converters));
     nargs = PyTuple_GET_SIZE(converters);
-    /* Hm. What to return in case of error?
-       For COM, 0xFFFFFFFF seems better than 0.
-    */
     if (nargs < 0) {
         PrintError("BUG: PySequence_Length");
         goto Done;
     }
 
-    PyObject *args_stack[CTYPES_MAX_ARGCOUNT];
-    if (nargs <= CTYPES_MAX_ARGCOUNT) {
+    PyObject *args_stack[_PY_FASTCALL_SMALL_STACK];
+    if (nargs <= _PY_FASTCALL_SMALL_STACK) {
         args = args_stack;
     }
     else {
-        args = PyMem_Malloc(nargs * sizeof(PyObject *));
+        args = (PyObject **)alloca(nargs * sizeof(PyObject *));
```

Comment:
Is an explicit cast from ``(void*)`` to ``(PyObject**)`` really needed? (I suggest to remove it).

Reply:
It's an age-old debate among C programmers ;)

https://en.wikipedia.org/wiki/C_dynamic_memory_allocation#Advantages_to_casting

I tend to agree to go without the cast.